redaxo/src/core/lib/request.php

```diff
@@ -257,7 +257,7 @@ public static function isPJAXContainer($containerId)
             return false;
         }
 
-        return isset($_SERVER['HTTP_X_PJAX_CONTAINER']) && $_SERVER['HTTP_X_PJAX_CONTAINER'] == $containerId;
+        return $containerId === rex::getRequest()->headers->get('X-Pjax-Container');
```

Comment:
Hier hast du das casing des headers verändert.. weiter oben in isPjaxRequest nicht?

Reply:
symfony normalisiert es intern sowieso. Aber ich werde eins von beiden anpassen, sodass es bei uns einheitlich ist.